src/sentry/integrations/msteams/integration.py

```diff
@@ -38,10 +39,13 @@
 ]
 
 
-INSTALL_NOTICE_TEXT = "INSTALL TEXT"
+INSTALL_NOTICE_TEXT = (
+    "Visit the Teams Marketplace to install this integration. After adding the integration"
+    " to your team, you will get a message in the General channel to complete installation."
+)
 
 external_install = {
-    "url": u"https://google.com",  # TODO: set correct URL
+    "url": u"https://teams.microsoft.com/l/app/%s" % options.get("msteams.app-id"),
```

Comment:
@scefali Apparently using `format` is preferrable to `%` bc the modulo operator for string formatting is [considered obsolete and may not be supported in future versions of Python](https://docs.python.org/release/3.1.5/library/stdtypes.html#old-string-formatting-operations).